tests/drivers/virtualization/ivshmem/doorbell/boards/qemu_kvm_arm64.conf

```diff
@@ -0,0 +1,19 @@
+CONFIG_PCIE_CONTROLLER=y
+CONFIG_PCIE_ECAM=y
+
+# Hungry PCI requires at least 256M of virtual space
+CONFIG_KERNEL_VM_SIZE=0x80000000
+
+# Hungry PCI requires phys addresses with more than 32 bits
+CONFIG_ARM64_VA_BITS_40=y
+CONFIG_ARM64_PA_BITS_40=y
+
+# MSI support requires ITS
+CONFIG_GIC_V3_ITS=y
+
+# ITS, in turn, requires dynamic memory (9x64 + alignment constrains)
+# Additionally, our test also uses malloc
+CONFIG_HEAP_MEM_POOL_SIZE=1048576
+
+# memory for userspace test
+CONFIG_MINIMAL_LIBC_MALLOC_ARENA_SIZE=8194304
```

Comment:
Just curious about this number because it cannot be divided by 1024. So where does this number comes from? 👀

Reply:
Hello @SgrrZhf. Thank you for the review.

Yes, my bad... I forgot to correct this. The story of this funny value is this: I first tried 4MB (the default ivshmem shared memory size) for malloc arena (i.e., `4194304`); but this is not enough. So, I wanted to go to the next power-of-two (8MB)... but I only changed the first digit (`4` -> `8`) and forgot to correct the rest. I will update and add a little explanation